addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -650,22 +649,36 @@ private static AlertNode getAlertNode(String name, String desc, int risk, int co
                 uriStr,
                 "Test Param",
                 "Test \"Attack\\\"",
-                "Test 'Other\\",
+                "Test " + prefix + "'Other\\",
                 "Test Solution",
                 "Test Reference",
                 "Test <p>Evidence",
                 123,
                 456,
                 msg);
+
         Map<String, String> tags = new HashMap<>();
         tags.put("tagkey", "tagvalue");
         alert.setTags(tags);
-        node.setUserObject(alert);
+        return alert;
+    }
+
+    private static AlertNode getAlertNode(String name, String desc, int risk, int confidence)
+            throws URIException, HttpMalformedHeaderException {
+        AlertNode node = new AlertNode(risk, name);
+        Alert alert1 = createAlertNode(name, desc, risk, confidence, "");
+        Alert alert2 = createAlertNode(name, desc, risk, confidence, "Another ");
+        node.setUserObject(alert1);
+        node.setUserObject(alert2);
```

Comment:
This does not look right.

Reply:
The method or the way of writing ?

Reply:
Calling it twice, the second call overrides the first.

Reply:
I think only one of these statements should be written

Reply:
Right.